src/main/java/seedu/taskman/model/TaskMan.java

```diff
@@ -81,9 +82,9 @@ public void resetData(ReadOnlyTaskMan newData) {
      *
      * @throws UniqueActivityList.DuplicateActivityException if an equivalent task already exists.
```

Comment:
Change task to event in comments. 

Same for the method below this. Change task to activity (?)